nautobot/core/api/utils.py

```diff
@@ -171,17 +171,19 @@ def get_view_name(view, suffix=None):
     """
     Derive the view name from its associated model, if it has one. Fall back to DRF's built-in `get_view_name`.
     """
-    if hasattr(view, "queryset"):
+    name = getattr(view, "name", None)
+    if name is not None:
+        return view.name
```

Comment:
I realize this is just moving existing code but it's a weird bit of logic. Why not:

```suggestion
    if hasattr(view, "name") and view.name:
        return view.name
```

Reply:
I made that update. I also looked at the latest DRF built-in method and updated some of the logic to match what they use now. From what I can tell, the initial code in Netbox/Nautobot was from a commit 11 years ago. There was a change 6 years ago that made some updates/improvements